src/main/groovy/io/saagie/plugin/dataops/models/JobVersionDTO.groovy

```diff
@@ -33,7 +33,11 @@ class JobVersionDTO implements IExists, Comparable{
         commandLine = version.commandLine
         runtimeVersion = version.runtimeVersion
         releaseNote = version.releaseNote
-
+    
+        if(current.number) {
+            number = current.number
```

Comment:
number? what is this?
current? what is this?

Reply:
current is job v1 map object from version 1 api.
Number is property of job v1